src/Backend/Modules/Blog/Actions/Add.php

```diff
@@ -62,7 +62,7 @@ private function loadForm(): void
         $this->form->addText('title', null, null, 'form-control title', 'form-control danger title');
         $this->form->addEditor('text');
         $this->form->addEditor('introduction');
-        $this->form->addRadiobutton('hidden', $rbtHiddenValues, false);
+        $this->form->addRadiobutton('hidden', $rbtHiddenValues, 0);
```

Comment:
While I was reviewing the release of spoon 3 I saw something that might have fixed this, I'll check this again